python/ray/tune/integration/wandb.py

```diff
@@ -198,12 +199,18 @@ def __init__(
         self.kwargs = kwargs
 
     def run(self):
-        # Since we're running in a separate process already, use threads.
+        wandb.require("service")
         wandb.init(*self.args, **self.kwargs)
+        wandb.setup()
         while True:
             result = self.queue.get()
```

Comment:
Can we add some comments about what can be the stuff circulated through this queue?